src/modules/MeasureTool/MeasureToolUI/MainWindow.xaml

```diff
@@ -261,13 +261,21 @@
                 Click="BoundsTool_Click"
                 Content="&#xEF20;"
                 Style="{StaticResource ToggleButtonRadioButtonStyle}"
-                ToolTipService.ToolTip="{x:Bind p:Resources.Bounds}" />
+                KeyboardAcceleratorPlacementMode="Auto"
+                ToolTipService.ToolTip="{x:Bind p:Resources.Bounds}">
+                <ToggleButton.KeyboardAccelerators>
+                    <KeyboardAccelerator Key="Number1" Modifiers="Control" Invoked="KeyboardAccelerator_Invoked"/>
```

Comment:
This is a nice feature, it should be documented somewhere.

Reply:
@jaimecbernardo do we open a PR for docs like that one https://github.com/MicrosoftDocs/windows-dev-docs/pull/4020 or request the docs team to update it?

Reply:
We open a PR and ask to merge it only after we release. Community members like @htcfreek have been doing it as well ;) 
Add Clint and me to the reviewers, please.

Reply:
nice, created an issue with "Help wanted" label